torch/csrc/autograd/profiler.h

```diff
@@ -51,7 +52,7 @@ struct Event {
     if(record_cuda) {
       TORCH_CUDA_CHECK(cudaGetDevice(&device_));
       TORCH_CUDA_CHECK(cudaEventCreate(&event));
-      auto stream = at::globalContext().getCurrentCUDAStream();
+      auto stream = ::at::cuda::globalCUDAContext().getCurrentCUDAStream();
```

Comment:
Maybe this is blasphemy, but it seems like it would almost be OK to just stuff the most commonly used guys in the top level namespace, e.g., `at::cuda::getCurrentCUDAStream()`. Maybe not all of them, but the ones that are most commonly used.